### ISSUE TITLE
[Android] Nested route malformed URI parsing

### ISSUE THREAD

**Issue by Poweranimal** [lulupointu/vrouter#28]
Route URI parsing fails with nested routes when going back the tree.

**Setup**
Path: `/child/nested`
Hierachy: `LandingPage` -> `ChildPage` -> `NestedChildPage`

Going back from `NestedChildPage` to `ChildPage` fails to parse the URI.

**Error**
```
E/flutter ( 9996): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Exception: No route could be found for the url //child
E/flutter ( 9996): #0      VRouterState._updateUrl (package:vrouter/src/vrouter/vrouter.dart:898:9)
E/flutter ( 9996): #1      VRouterState._pop (package:vrouter/src/vrouter/vrouter.dart:1267:7)
E/flutter ( 9996): <asynchronous suspension>
```
**Example**
<details>
  <summary>Code</summary>

```dart
import 'package:flutter/material.dart';
import 'package:vrouter/vrouter.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return VRouter(
        routes: [
          VWidget(path: '/', widget: LandingPage(), stackedRoutes: [
            VWidget(path: 'child', widget: ChildPage(), stackedRoutes: [
              VWidget(path: 'nested', widget: NestedChildPage())
            ])
          ])
        ],
    );
  }
}

class LandingPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('LANDING PAGE')),
      body: Center(
        child: ElevatedButton(
          onPressed: () => context.vRouter.push('/child'),
          child: Text('SHOW CHILD')
        ),
      ),
    );
  }
}

class ChildPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('CHILD PAGE')),
      body: Center(
        child: ElevatedButton(
            onPressed: () => context.vRouter.push('/child/nested'),
            child: Text('SHOW NESTED CHILD')
        ),
      ),
    );
  }
}

class NestedChildPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('NESTED CHILD PAGE')),
      body: Center(
        child: Text('Hello World'),
      ),
    );
  }
}
```
</details>

**Device**
    OS:           Android
    Version:    30
    Device:     Pixel 4
    CPU/ABI:  x86_64
    Emulator:  true

**Comment by lulupointu** [comment]
Mmmm, I know why, let me fix that and come back to you. Thanks for pointing that out.

**Comment by lulupointu** [comment]
Solved in v1.1.0+12 :rocket:

**Comment by Poweranimal** [comment]
> Thanks for the well written issue btw, a pleasure to work with you :wink: 

You're welcome.
I'm happy to contribute to your project 😊
